bot/src/services/supreme/CheckoutService.js

```diff
@@ -1,10 +1,10 @@
 const ignoredIds = ['g-recaptcha-response', 'number_v', 'order_billing_address_3'];
 
-export default class CheckoutService {
-  static processFields(inputs, settings) {
+export default class CheckoutService{
+  static processFields(inputs, settings, delay) {
     const successes = [];
     for (let i = 0; i < inputs.length; i += 1) {
-      const success = this.processField(inputs[i], settings);
+      const success = this.processField(inputs[i], settings, delay);
```

Comment:
`this.processField` returns a promise, so it should be `const success = await this.processField(inputs[i], settings, delay);`